docs/development/pigs/pig-001.rst

```diff
@@ -0,0 +1,120 @@
+  are aware for things that might be controversial and have long-term
+  effects for Gammapy, e.g. when to drop Python 2 support or defining
+  a release cycle for Gammapy. These PIGs can usually be very short, a page or less.
+
+Writing a PIG
+=============
+
+Anyone is welcome to write a PIG!
+
+PIGs are written as RST files in the ``docs/development/pigs`` folder in the
+main Gammapy repository, and submitted as pull requests.
+
+Most discussions concerning Gammapy will happen by talking to each other directly
+(calls or face-to-face), or online on the mailing list or Github.
+If you're not sure if you should write a PIG, please don't! Instead bring the
+topic up in discussions first with other Gammapy developers, or on the mailing list,
+to get some initial feedback. This will let you figure out if writing a PIG
+will be helpful or not to realise your proposal.
+
+For now, we don't require a formal structure, apart from adding the author,
+creation date and status points at the top, as done at the top of this PIG.
+Please make your proposal clearly and keep the initial proposal short.
+If more information is needed, people that will review it will ask for it.
+
+Writing a PIG doesn't mean you have to implement it. That said, we expect that
+most PIGs will propose something that requires developments where someone has
+the intention to implement it within the near future (say within the next year).
+But it's not required, e.g. if you have a great idea or vision for Gammapy
+that requires a lot of development, without the manpower to execute the idea,
+writing a PIG could be a nice way to share this idea in some detail, with the
+hope that in collaboration with others it can eventually be realised.
+
+PIG review
+==========
+
+PIG review happens on the pull request on Github.
+
+When a PIG is put up, an announcement with a link to the pull request should
+be sent both to the Gammapy mailing list and the Gammapy coordinator list.
+
+Anyone is welcome to review it and is encouraged to share their thoughts
+in the discussion!
+
+Please note that Github hides inline comments after they have been edited,
+so we suggest that you use inline comments for minor points like spelling mistakes only.
+Put your main feedback as normal comments in the "Conversation" tab,
+so that for someone reading the discussion later they will see your comment directly.
+
+The final decision on any PIG is made by the Gammapy coordination committee.
+We expect that in most cases, the people participating in the PIG review will
+reach a consensus and the coordination committee will simply follow the outcome
+of the public discussion. But in unusual cases where disagreement remains,
+the coordination committee will talk to the people involved in the discussion
+with the goal to reach consensus or compromise, and then make the final decision.
+
+PIG status
+==========
+
+PIGs can have a status of:
+
+* "draft" - in draft status, either in the writing or dicussion phase
```

Comment:
Are there formal criteria when the PR with the PIG is merged? Is it kept open until the decision is made? Then a "draft" status would not be needed...

Reply:
I think there will be cases where the PIG PR is merged, if it just proposes a well-thought out plan. And there's other cases where someone puts a very preliminary idea, then there is some discussion, some prototyping, and the PR stays open for months. This has happened for APEs and PEPs and will happen to us, and I think that's OK. So in that phase, the status "draft" is appropriate. The status names are taken from PEPs, see PEP 1. OK?

Reply:
I've added a section here explaining a little more about the status:

> When a PIG is put up for discussion as a pull request, it should have a status of
"draft". Then once the discussion and review is done, the status will change to
one of "withdrawn", "accepted" or "rejected". The reviewers should add a section
"Decision rationale" with a sentence or paragraph summarising the discussion and
decision on this PIG. Then in any case, the PIG should be merged, even if it's
status is "withdrawn" or "rejected".

@adonath - I think your question about decision making and when a PIG should be merged are already addressed in the paragraphs "PIG review" and "Final remarks" that come before / after this "PIG status" section.